src/navigation/Navigator.tsx

```diff
@@ -43,6 +44,7 @@ const HomeStackNavigator = (): JSX.Element | null => {
           initialParams={{ initialSelection: true }}
           options={({ navigation }) => options(manageSelection, navigation)}
         />
+        <Stack.Screen name='ManageSelection' component={ManageSelectionsScreen} />
```

Comment:
Why have you added this here? Isn't it enough to keep it in the HomeScreenNavigator as before?

Reply:
No, I then can't actually navigate in there when opening the app after a fresh installation.
<img width="378" alt="image" src="https://github.com/digitalfabrik/lunes-app/assets/18513943/259cc9e5-9b06-4027-96ce-c4fbda70d55d">

Reply:
@LeandraH do you have any clue why this wasn't needed before?

Reply:
I assume it's needed now because with this PR you can navigate to the `ManageSelectionsScreen` from the `ScopeSelectionScreen`, and that one is in the `Navigator` and not in the `HomeStackNavigator`
https://github.com/digitalfabrik/lunes-app/pull/1003/files#diff-056a90d9f549141742c1c899d05ac3a087ef8b87b4f1fa7cdccee90d48bd9fa6R147